### PR TITLE
[Hackathon 7th] fix aishell3/vctk vc0/ernie

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR types

Others 
### PR changes

Others 
### Describe

Others

### CODE REVIEW

File: paddlespeech/t2s/modules/losses.py [hunk 1]
```diff
@@ -1115,7 +1115,8 @@ def forward(
                     paddle.reshape(xs_pad, (-1, self.odim))),
                 axis=-1)
         mlm_loss = paddle.sum((loss * paddle.reshape(
-            mlm_loss_pos, [-1]))) / paddle.sum((mlm_loss_pos) + 1e-10)
+            mlm_loss_pos,
+            [-1]).astype(loss.dtype))) / paddle.sum((mlm_loss_pos) + 1e-10)
```
Comment (zxcd) [inline paddlespeech/t2s/modules/losses.py:1119]: what's the data format of loss and mlm_loss_pos? `paddle.sum((mlm_loss_pos) + 1e-10)` don't need a `.astype`?

File: paddlespeech/t2s/modules/tacotron2/attentions.py [hunk 2]
```diff
@@ -171,7 +171,9 @@ def forward(
         if paddle.sum(att_prev) == 0:
             # if no bias, 0 0-pad goes 0
             att_prev = 1.0 - make_pad_mask(enc_hs_len)
-            att_prev = att_prev / enc_hs_len.unsqueeze(-1)
+            unsqueeze_enc_hs_len = enc_hs_len.unsqueeze(-1).astype(
```
Comment (zxcd) [inline paddlespeech/t2s/modules/tacotron2/attentions.py]: why add new line here?

File: paddlespeech/t2s/modules/losses.py (hunk 1 above)
Comment (Liyulingyue) [reply paddlespeech/t2s/modules/losses.py:1119]: mlm_loss_pos 大概是个 bool

File: paddlespeech/t2s/modules/tacotron2/attentions.py [hunk 3]
```diff
@@ -171,7 +171,9 @@ def forward(
...
+                att_prev.dtype)
+            att_prev = att_prev / unsqueeze_enc_hs_len
```
Comment (Liyulingyue) [inline paddlespeech/t2s/modules/tacotron2/attentions.py]:
```suggestion
            att_prev = att_prev / enc_hs_len.unsqueeze(-1).astype(att_prev.dtype)
```

File: paddlespeech/t2s/modules/tacotron2/attentions.py [hunk 4]
```diff
@@ -171,7 +171,7 @@ def forward(
         if paddle.sum(att_prev) == 0:
             # if no bias, 0 0-pad goes 0
             att_prev = 1.0 - make_pad_mask(enc_hs_len)
-            att_prev = att_prev / enc_hs_len.unsqueeze(-1)
+            att_prev = att_prev / enc_hs_len.unsqueeze(-1).astype(att_prev.dtype)
```
Comment (Liyulingyue) [inline paddlespeech/t2s/modules/tacotron2/attentions.py]:
```suggestion
            att_prev = att_prev / enc_hs_len.unsqueeze(-1).astype(
                att_prev.dtype)
```

**Review by zxcd — APPROVED** [review]
LGTM
